terraform/modules/google-cloud/apps/relay/main.tf

```diff
@@ -147,7 +181,7 @@ resource "google_compute_instance_template" "application" {
 
   project = var.project_id
 
-  name_prefix = "${local.application_name}-template-${each.key}-"
+  name_prefix = "${local.application_name}-${each.key}-"
```

Comment:
We'll probably need to change this as well so that GCP creates a new set of instances instead of trying to keep the old instance groups around.